feature/bgp/multipath/otg_tests/bgp_multipath_wecmp_lbw_community_test/bgp_multipath_wecmp_test.go

```diff
@@ -71,13 +71,32 @@ func configureOTG(t *testing.T, bs *cfgplugins.BGPSession) {
 		routeAddress.SetPrefix(prefixP4Len)
 		routeAddress.SetCount(prefixesCount)
 		bgp4PeerRoute.AddPath().SetPathId(pathID)
-		bgpExtCom := bgp4PeerRoute.ExtendedCommunities().Add()
-		bgpExtCom.NonTransitive2OctetAsType().LinkBandwidthSubtype().SetBandwidth(float32(linkBw[i-2] * 1000))
 	}
 
 	configureFlow(bs)
 }
 
+func configureInternalNetwork(t *testing.T, ate *ondatra.ATEDevice, top gosnappi.Config) {
```

Comment:
consider making this a library and moving to internal/otgutils

Reply:
it was a typo, I've changed the name of this method
This method is attaching LB ext community to internal network.